Sources/SizeCalculable.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  Sizable.swift
+//  PinLayout-iOS
+//
+//  Created by Antoine Lamy on 2018-06-12.
+//  Copyright © 2018 mcswiftlayyout.mirego.com. All rights reserved.
+//
+
+import Foundation
+
+@available(OSX 10.10, *)
```

Comment:
The `@available` is not really required in the protocol declaration.

Reply:
Removed